packages/synapse-react-client/src/components/ReleaseCard/ReleaseCard.test.tsx

```diff
@@ -111,14 +110,16 @@ const defaultReleaseCardLargeProps: ReleaseCardLargeProps = {
 }
 
 function renderComponent(props: ReleaseCardProps) {
-  return render(
-    <Router history={createMemoryHistory()}>
-      <ReleaseCard {...props} />
-    </Router>,
+  const router = createMemoryRouter([
     {
-      wrapper: createWrapper(),
+      path: '/',
+      element: <ReleaseCard {...props} />,
     },
-  )
+  ])
```

Comment:
In tests that rely on a router, we can use `createMemoryRouter` and pass it to a `RouterProvider` component